Langs/0422.lang

```diff
@@ -28,9 +29,9 @@ Storage Available=Доступный об'єм
 TPM Version=Версія TPM
 Checking...=Йде перевірка...
 64 Bit CPU=64-бітний процессор
-64 Bit OS=64-бітна ОС
+64 Bit OS=64-бітна Операційна система
```

Comment:
Did you check that it fits? Feel free to correct later if it seems too long.

Reply:
> Did you check that it fits? Feel free to correct later if it seems too long.

Thanks for reminding me, I'll be sure to check it out.

Reply:
> Did you check that it fits? Feel free to correct later if it seems too long.

@micwoj92 

<img width="853" alt="Screen Shot 2021-07-11 at 21 01 00" src="https://user-images.githubusercontent.com/62497891/125207460-c1d20f00-e28c-11eb-8230-98372dc91755.png">

Reply:
Thx for checking.